### PR TITLE
Code Exchange Upgrade

### LINKED ISSUE #59: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] Twilio [LICENSE](https://github.com/twilio-labs/code-exchange/blob/master/LICENSE) exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct
- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by po5i**
- Use Laravel sample template
- Upgrade dependencies
- Replace Travis with Github Actions
- Add Makefile
- Add Docker
- Resolves #59

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Sample App powered by Twilio",
+  "description": "A template for getting started with Twilio and Laravel/PHP.",
```
Comment (maylonpedroso) [inline app.json]: This should be "Appointment Reminders" app name and description

File: public/css/app.css [hunk 2]
```diff
@@ -1,8 +1,10893 @@
-@import url(https://fonts.googleapis.com/css?family=Nunito);
+@import url(https://fonts.googleapis.com/css?family=Nunito);@charset "UTF-8";
```
Comment (maylonpedroso) [inline public/css/app.css]: Isn't this file built by laravel mix. It should be ignored by git. Same with `public/js/app.js` and `public/js/date-time-picker.js`?

File: webpack.mix.js [hunk 3]
```diff
@@ -11,5 +11,5 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
-   .sass('resources/sass/app.scss', 'public/css');
+mix.js('resources/js/datetime-picker.js', 'public/js')
+    .sass('resources/sass/app.scss', 'public/css');
```
Comment (maylonpedroso) [inline webpack.mix.js]: The `app.js` should be included too.

**Review by maylonpedroso — CHANGES_REQUESTED** [review]

File: public/css/app.css (hunk 2 above)
Comment (po5i) [reply public/css/app.css]: Good catch. Updated.

**Review by maylonpedroso — APPROVED** [review]
:+1:

**Pull request merged**